sqlx-core/src/query.rs

```diff
@@ -51,6 +51,10 @@ where
     fn into_parts(self) -> (&'q str, Option<DB::Arguments>) {
         (self.query, Some(self.arguments))
     }
+
+    fn query_string(&self) -> &'q str {
```

Comment:
For now, can we add `#[doc(hidden)]` on `query_string` (needs to be in the trait _and_ every impl) ?

Reply:
Yes. I was hoping for a way to do this without adding or modifying this trait, but since `Execute` doesn't require `Clone`, and `into_parts` moves `self`, I didn't see a way to do that.